src/js/components/footer.js

```diff
@@ -0,0 +1,39 @@
+import { newText, newContainer, newMultimedia } from './helpers';
+
+const createLInks = (arraySocial) => {
```

Comment:
Please rename the function to be `createLinks` so that it uses camel case notation correctly

Reply:
Done